src/coffea/dataset_tools/dataset_query.py

```diff
@@ -553,7 +571,11 @@ def load_dataset_definition(
         and selected results and replicas following the options.
 
         - query_results_strategy:  "all" or "manual" to be prompt for selection
-        - replicas_strategy: "round-robin", "choose" (to manually choose the sites), "manual": to be prompt for manual decision case by case
+        - replicas_strategy:
+            - "round-robin": select randomly from the available sites for each file
+            - "choose": filter the sites with a list of indices for all the files
+            - "first": take the first risult returned by rucio
```

Comment:
```suggestion
            - "first": take the first result returned by rucio
```

Reply:
Just saw this typo while looking at the changes, will give it a spin later. Thanks @valsdav!